index.bs

```diff
@@ -486,6 +486,36 @@ reasonable future change. We could run [=process a network cookie access for bou
 in the <a spec="fetch">HTTP-network-or-cache fetch</a> algorithm after step 8.21.1.2,
 "... [=append=] (`Cookie`, cookies) to httpRequest’s [=header list=]. ..."
 
+<h5 id="bounce-tracking-mitigations-storage-access-monkey-patch">Storage Access Monkey Patch</h5>
+
+Each [=top-level traversable=] maintains a record of which sites have accessed storage in the current [=extended navigation=].
+
+Insert the following steps in the <a spec="storage">obtain a storage key</a> algorithm before step 4, "Return key":
+
+1. Run [=process a storage access for bounce tracking mitigations=] given <var ignore>environment</var>.
+
+Issue: TODO: Verify that this algorithm is the correct location to catch precisely those sites which access non-cookie storage.
```

Comment:
Done. Changed to a Note and updated the comment.

Note that I had to change to the <p class> syntax since it doesn't look like linking the NOTE text macro is supported.